src/Adapter/Customer/Repository/CustomerRepository.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * 2007-2020 PrestaShop SA and Contributors
```

Comment:
"Old style" license headers 😝  we now use `Copyright since 2007 PrestaShop SA and Contributors`

Reply:
Riight. Must have copied them from somewhere. Will fix!